### DISCUSSION TITLE
[Documentation / Overview] Please consider providing an overview in the README as table

### DISCUSSION THREAD

**Discussion by rubyFeedback** [DustinBrett/daedalOS#46] [Feature Requests]
Hello there Dustin,

If you have some spare time at some coming weekend, would it be possible to add a table overview
in the main README, that briefly mentions / points at available functionality?

Something where people can look at a HTML table and quickly see, "ok, two PDF readers are available,
let's see, word processor, games" etc... - something for people to quickly see what is available and
possible.

You currently provide a semi-overview with some details, but I think a short, succint table in addition
to that would be super-helpful, so that people can see what is possible on a first glance. Anyway, please
feel free to ignore / disregard this issue here at any moment in time!

**Comment by DustinBrett** [comment]
Thanks for the feedback. I will for sure consider this idea and some way to see what daedalOS can do at more of a glance. Thanks
